### PR TITLE
709 -  Question Confirmation

### PR DESCRIPTION
**Pull request by tgandrews**
### What is the context of this PR?
Add the ability to CRUD question confirmation against a page.

### Note
Piping is disabled as this is awaiting the ability to filter the page previous answers/metadata.

### How to review 
1. Ensure you can create, duplicate, update, delete and restore a question confirmation against a question page.

### Dependencies
- [ ] - https://github.com/ONSdigital/eq-author-api/pull/180

### CODE REVIEW

**Comment by tgandrews** [comment]
[Playback confirmation](https://trello.com/c/WniDYUci/709-playback-confirmation)

**Comment by adammahmood** [comment]
The copy is different from the designs in Trello.  Is this intended? Also, can we capitalize these options in a consistent way? 

screenshot 2018-11-15 at 13 18 58

**Comment by adammahmood** [comment]
These radio options don't match the design specs. Is this intended?

screenshot 2018-11-15 at 13 19 59

**Comment by adammahmood** [comment]
There seems to be more padding than design specs between the delete button and label. Is this intended?

screenshot 2018-11-15 at 13 21 03

**Comment by adammahmood** [comment]
Shall we change this copy to mention confirmation pages?
screenshot 2018-11-15 at 13 23 32

**Comment by tgandrews** [comment]
As discussed the prototype and design specs differ quite a bit and this was done against the prototype which IIRC was discussed in the planning and refinement.

File: src/components/Tabs/index.js [hunk 1]
```diff
@@ -52,21 +53,34 @@ const DisabledTab = styled(Tab.withComponent("span"))`
   color: ${colors.lightGrey};
 `;
 
+const buildUrl = match => {
```
[the remark this answers is not in the document]
Comment (tgandrews) [reply src/components/Tabs/index.js]: Done. If happy please resolve.

File: src/components/QuestionConfirmationRoute/Editor.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+`;
+
+const getFirstAnswer = get("answers[0]");
+export const MultipleChoiceBlock = ({ page }) => {
+  const answer = getFirstAnswer(page);
+  if (!answer || !isMultipleChoice(answer.type)) {
+    return null;
+  }
+  return (
+    <MCBWrapper>
```
[the remark this answers is not in the document]
Comment (tgandrews) [reply src/components/QuestionConfirmationRoute/Editor.js]: Extracted MCB to its own file - which removes the need for these prefixes. Please resolve if you are happy.

**Comment by adammahmood** [comment]
Approving this PR.  I'll leave final design tweaks for Hamish to approve 👍

**Comment by jamesnunn** [comment]
[Prevent session-expiry end point extending the session expiry (1SP)](https://trello.com/c/5BT1W6om/4957-prevent-session-expiry-end-point-extending-the-session-expiry-1sp)

**Pull request merged**